Playmate/Models/Session.m

```diff
@@ -22,10 +22,17 @@ @implementation Session
 @dynamic occupied;
 
 + (nonnull NSString *)parseClassName {
-    return @"SportsSession";
+    return [@"Sports" stringByAppendingString:NSStringFromClass([Session class])];
```

Comment:
Understood, I also wished that it could be the same name, but Parse database has a default class named Session to keep track of user logins, etc. so I could not use the same name